configs/Pico/BoardConfig.h

```diff
@@ -220,6 +220,20 @@
 #define DUAL_DIRECTIONAL_STICK_MODE DPAD_MODE_DIGITAL
 #define DUAL_DIRECTIONAL_COMBINE_MODE DUAL_COMBINE_MODE_MIXED
 
+// TILTAdd-on Options
```

Comment:
There are fallbacks for each of the Tilt function defines in `tilt.h`, so they aren't needed in every `BoardConfig.h` file. Leave the defines here in the `Pico/BoardConfig.h`, but please remove from all other `BoardConfig.h` files.